features/features.json

```diff
@@ -1,4 +1,17 @@
  [
+  {
+    "title": "Direct image uploads in the forums",
+    "description": "Allows you to easily upload images to the Scratch forums for posts and signatures without having to use third party image uploading services.",
+    "credits": ["ISTILLMAKESTUFF", "TheGlassPenguin", "StickFireGames"],
+    "urls": [
+      "https://scratch.mit.edu/users/ISTILLMAKESTUFF",
+      "https://scratch.mit.edu/users/TheGlassPenguin",
+      "https://scratch.mit.edu/users/stickfiregames"
+    ],
+    "file": "upload-img-directly",
+    "type": ["Website"],
+    "tags": ["New", "Recommended", "Forums"]
```

Comment:
The "Forums" tag doesn't do anything. We just need "New" and "Recommended".

Reply:
done